source/precalculus/source/08-TE/03.ptx

```diff
@@ -12,6 +12,1000 @@
+        </p>
+      </hint>
+      <answer>
+      <p>
+      <m>\theta=0, \frac{2\pi}{3}, \frac{4\pi}{3}</m>
+      </p>
+      </answer>
+      </task>
+    </activity>
+
+    <remark>
+      <p>
+        In some cases, it is not easy to isolate the trigonometric term without having to use some other strategies we have learned. The next few activities will help us learn how to apply methods we have used before to solve trigonometric equations.
+      </p>
+    </remark>
+
+    <activity xml:id="trig-square-root-property">
+      <introduction>
+        <p>
+          Consider the equation <m>4\sin^2\theta-2=0</m>.
+        </p>
+      </introduction>
+
+      <task>
+        <statement>
+          <p>
+            Use algebra to isolate the <m>\sin^2\theta</m> term.
+          </p>
+        </statement>
+        <answer>
+          <p>
+            <m>\sin^2\theta=\frac{1}{2}</m>
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            To undo a squared term, you will need to take the square root (recall square root property). Take the square root of <m>\sin^2\theta</m> to get <m>\sin\theta</m>. What does <m>\sin\theta</m> equal?
+            <ol marker= "A." cols="4">
+            <li> <m>\pm\sqrt{\frac{1}{2}}</m> </li>
+            <li> <m>\pm\frac{1}{\sqrt{2}}</m> </li>
+            <li> <m>\pm\frac{1}{2}</m> </li>
+            <li> <m>\pm\frac{\sqrt{2}}{2}</m> </li></ol>
+          </p>
+        </statement>
+        <answer>
+          <p>
+            A, B, and D
+          </p>
+        </answer>
+      </task>
+  
+      <task>
+        <statement>
+         <p>
+          Although there are multiple ways to represent <m>\sin\theta</m>, which solution(s) from part (b) do you see on the unit circle?
+          <ol marker= "A." cols="4">
+            <li> <m>\pm\sqrt{\frac{1}{2}}</m> </li>
+            <li> <m>\pm\frac{1}{\sqrt{2}}</m> </li>
+            <li> <m>\pm\frac{1}{2}</m> </li>
+            <li> <m>\pm\frac{\sqrt{2}}{2}</m> </li></ol>
+         </p>
+        </statement>
+        <answer>
+          <p>
+            D
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            Notice there are two solutions <m>\sin\theta</m>, which corresponds to <m>4</m> values of <m>\theta</m> that satisfies the equation <m>\sin\theta=\pm\frac{\sqrt{2}}{2}</m>. What values of <m>\theta</m> satisfies that equation on the interval <m>[0, 2\pi)</m>?
+            <ol marker= "A." cols="3">
+            <li> <m>\frac{\pi}{6}</m> </li>
+            <li> <m>\frac{\pi}{3}</m> </li>
+            <li> <m>\frac{5\pi}{6}</m> </li>
+            <li> <m>\frac{2\pi}{3}</m> </li>
+            <li> <m>\frac{7\pi}{6}</m> </li>
+            <li> <m>\frac{11\pi}{6}</m> </li></ol>
+          </p>
+        </statement>
+        <answer>
+          <p>
+            A, C, E, and F
+          </p>
+        </answer>
```

Comment:
I don't see the correct answers among the answer choices, and thus the supplied answer is incorrect.